src/app/modules/data-set/nav/navbar.component.ts

```diff
@@ -0,0 +1,75 @@
+import { Component, OnInit } from '@angular/core';
+import { MatSlideToggleChange } from '@angular/material/slide-toggle';
+import { Router } from '@angular/router';
+import { SessionStorageService } from 'src/app/session-storage.service';
+import { DataSetProject } from '../model/data-set-project/data-set-project.model';
+import { DataSet } from '../model/data-set/data-set.model';
+import { Instance } from '../model/instance/instance.model';
+import { AnnotationNotificationService } from '../services/annotation-notification.service';
+
+
+@Component({
+  selector: 'de-navbar',
+  templateUrl: './navbar.component.html',
+  styleUrls: [ './navbar.component.css' ]
+})
+export class NavbarComponent implements OnInit {
+
+  public chosenDataset: DataSet = new DataSet();
+  public chosenProject: DataSetProject = new DataSetProject();
+  public chosenInstance: Instance = new Instance(this.sessionService);
+  public annotationsCounter: boolean = false;
+
+  constructor(private sessionService: SessionStorageService, private router: Router,
+    private annotationNotificationService: AnnotationNotificationService) {}
+
+  public ngOnInit(): void {
+      if (!this.isLoggedIn())this.router.navigate(['/login']);
```

Comment:
Will this always work? What if I log out, then click Back?

Reply:
I will fix this in the new PR related to login.